libp2p/src/lib.rs

```diff
@@ -188,8 +190,7 @@ pub use libp2p_identity::PeerId;
     ),
     feature = "websocket",
     feature = "noise",
-    feature = "mplex",
-    feature = "yamux"
+    feature = "yamux",
```

Comment:
Same here, this needs to be restored for backwards compatibility.

Reply:
Done